hugo.toml

```diff
@@ -8,6 +8,7 @@ theme = ["docsy"]
 contentDir = "content/en"
 defaultContentLanguage = "en"
 defaultContentLanguageInSubdir = false
+i18nDir = "./data/i18n"
```

Comment:
This is moved under language settings as the script was not able to catch the directory, (All build settings remain fully functional)